interface/wx/grid.h

```diff
@@ -4565,6 +4608,13 @@ class wxGrid : public wxScrolledWindow
     //@}
 
 
+    void DrawGridCellArea( wxDC& dc , const wxGridCellCoordsArray& cells );
```

Comment:
I don't think we want to make all these functions part of our public API, I have no idea why are they `public` in the first place...